libraries/Microsoft.Bot.Connector/OAuthApiClient.cs

```diff
@@ -19,7 +19,7 @@ namespace Microsoft.Bot.Connector
     /// <summary>
     /// Service client to handle requests to the Bot Framework API service.
     /// </summary>
-    public class OAuthClient : ServiceClient<OAuthClient>
```

Comment:
I don't think we can take this, as this is a breaking change. Can we lave this "as-is" and instead introduce OAuthClient2?